commit 2d040ce2c7fd328b4e8b0c2363ebe0c2ea2cbc9f
Author: Mark Wielaard <mark@klomp.org>
Date:   Sat Jan 25 15:10:22 2020 +0100

    arm: Hook up preadv2 and pwritev2 syscalls.
    
    They work the same as on other linux architectures.

diff --git a/coregrind/m_syswrap/syswrap-arm-linux.c b/coregrind/m_syswrap/syswrap-arm-linux.c
index 93ebc3f4d..18468f07e 100644
--- a/coregrind/m_syswrap/syswrap-arm-linux.c
+++ b/coregrind/m_syswrap/syswrap-arm-linux.c
@@ -1016,6 +1016,8 @@ static SyscallTableEntry syscall_main_table[] = {
 
    LINX_(__NR_membarrier,        sys_membarrier),       // 389
    LINX_(__NR_copy_file_range,   sys_copy_file_range),  // 391
+   LINXY(__NR_preadv2,           sys_preadv2),          // 392
+   LINX_(__NR_pwritev2,          sys_pwritev2),         // 393
 
    LINXY(__NR_statx,             sys_statx),            // 397
 
